traffic_portal/app/src/traffic_portal_properties.json

```diff
@@ -189,6 +189,10 @@
             "title": "Check Path",
             "desc": "A path (ex: /crossdomain.xml) to verify the connection to the origin server with. This can be used by Check Extension scripts to do periodic health checks against the delivery service."
           },
+          "consistentHashRegex": {
+            "title": "Consistent Hash Regex",
+            "desc": "Regex used to extract parts of the request path using grouping in order to build a consistent request string to be used for cache selection. If this field is set on a steering delivery service, it will override the regex set on the target delivery services."
```

Comment:
is there a docs link you can link to kinda like the deepCachingType?

Reply:
yeah, I can add that